pinot-spi/src/main/java/org/apache/pinot/spi/utils/CommonConstants.java

```diff
@@ -699,6 +699,10 @@ public static class Server {
     public static final String CONFIG_OF_SEGMENT_STORE_URI = "segment.store.uri";
     public static final String CONFIG_OF_LOGGER_ROOT_DIR = "pinot.server.logger.root.dir";
 
+    public static final String CONFIG_OF_REALTIME_SEGMENT_CONSUMER_CLIENT_ID_SUFFIX =
+        "realtime.consumer.client.id.suffix";
```

Comment:
More concise since real-time is implicit for consumer
```suggestion
        "consumer.client.id.suffix";
```